### PR TITLE
Improve prize attribution

### PR DESCRIPTION
**Pull request by beasrodrigues24**
Why:
* It could be arrange to be instance and class methods

How:
* By creating a class method named random_select that returns a prize
* By creating an instance method named assign_prize that awards a lead a prize

### CODE REVIEW

**Comment by beasrodrigues24** [comment]
My question is: the sending email methods can also be arranged elsewhere? What else could be improved here?

File: app/models/lead.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
+# Class that holds lead information
 class Lead < ApplicationRecord
   validates :name, :email, presence: true
   belongs_to :prize, optional: true
+
+  def assign_prize(prize_won)
```
Comment (zamith) [inline app/models/lead.rb:8]: If it's this one line, why not call the update directly?

File: app/models/lead.rb [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
+# Class that holds lead information
```
Comment (zamith) [inline app/models/lead.rb:3]: Do you think this is an important comment?

Comment (beasrodrigues24) [reply app/models/lead.rb:3]: I was forced by rubocop

**Pull request merged**